packages/web-app-files/src/views/SharedWithMe.vue

```diff
@@ -162,26 +357,38 @@ export default {
       'UPDATE_RESOURCE'
     ]),
     ...mapMutations(['SET_QUOTA']),
-
+    setShowDeclined(value) {
+      showDeclined = value
+      this.$forceUpdate()
```

Comment:
Is the `this.$forceUpdate()` a mitigation for the change to `showDeclined` not being reactive / causing re-rendering of the view? If that's the case then it would be better to add `showDeclined` and `showAllPending` to the `data` prop after line 321. Then you don't need the getters and can skip the `this.$forceUpdate()` call. Same applies for the other setter.